--- /dev/null
+++ b/MODULE.bazel
@@ -0,0 +1,4 @@
+module(
+    name = "imgui",
+    version = "1.91.7-docking",
+)
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -0,0 +1,21 @@
+cc_library(
+    name = "imgui",
+    srcs = [
+        "imgui.cpp",
+        "imgui_demo.cpp",
+        "imgui_draw.cpp",
+        "imgui_tables.cpp",
+        "imgui_widgets.cpp",
+    ],
+    hdrs = [
+        "imconfig.h",
+        "imgui.h",
+        "imgui_internal.h",
+        "imstb_rectpack.h",
+        "imstb_textedit.h",
+        "imstb_truetype.h",
+    ],
+    includes = ["."],
+    strip_include_prefix = "",
+    visibility = ["//visibility:public"],
+)
--- a/imconfig.h
+++ b/imconfig.h
@@ -114,7 +114,7 @@
 // Your renderer backend will need to support it (most example renderer backends support both 16/32-bit indices).
 // Another way to allow large meshes while keeping 16-bit indices is to handle ImDrawCmd::VtxOffset in your renderer.
 // Read about ImGuiBackendFlags_RendererHasVtxOffset for details.
-//#define ImDrawIdx unsigned int
+#define ImDrawIdx unsigned int
 
 //---- Override ImDrawCallback signature (will need to modify renderer backends accordingly)
 //struct ImDrawList;
